motor/docstrings.py

```diff
@@ -1268,3 +1268,69 @@ async def coro():
 .. _$expr: https://docs.mongodb.com/manual/reference/operator/query/expr/
 .. _$where: https://docs.mongodb.com/manual/reference/operator/query/where/
 """
+
+create_data_key_doc = """Create and insert a new data key into the key vault collection.
+
+:Parameters:
+  - `kms_provider`: The KMS provider to use. Supported values are
+    "aws" and "local".
+  - `master_key`: Identifies a KMS-specific key used to encrypt the
+    new data key. If the kmsProvider is "local" the `master_key` is
+    not applicable and may be omitted.
+
+    If the `kms_provider` is "aws" it is required and has the
+    following fields::
+
+      - `region` (string): Required. The AWS region, e.g. "us-east-1".
+      - `key` (string): Required. The Amazon Resource Name (ARN) to
+         the AWS customer.
+      - `endpoint` (string): Optional. An alternate host to send KMS
+        requests to. May include port number, e.g.
+        "kms.us-east-1.amazonaws.com:443".
+
+    If the `kms_provider` is "azure" it is required and has the
+    following fields::
+
+      - `keyVaultEndpoint` (string): Required. Host with optional
+         port, e.g. "example.vault.azure.net".
+      - `keyName` (string): Required. Key name in the key vault.
+      - `keyVersion` (string): Optional. Version of the key to use.
+
+    If the `kms_provider` is "gcp" it is required and has the
+    following fields::
+
+      - `projectId` (string): Required. The Google cloud project ID.
+      - `location` (string): Required. The GCP location, e.g. "us-east1".
+      - `keyRing` (string): Required. Name of the key ring that contains
+        the key to use.
+      - `keyName` (string): Required. Name of the key to use.
+      - `keyVersion` (string): Optional. Version of the key to use.
+      - `endpoint` (string): Optional. Host with optional port.
+        Defaults to "cloudkms.googleapis.com".
+
+  - `key_alt_names` (optional): An optional list of string alternate
+    names used to reference a key. If a key is created with alternate
+    names, then encryption may refer to the key by the unique alternate
+    name instead of by ``key_id``. The following example shows creating
+    and referring to a data key by alternate name::
+
+      await client_encryption.create_data_key("local", keyAltNames=["name1"])
```

Comment:
So this is an exact clone of the sync dosctring except for this code example. I think we can probably just point the user to the sync API doc and also add this code sample to make this more maintainable.